packages/@aws-cdk/region-info/build-tools/static-data-facts/index.ts

```diff
@@ -0,0 +1,39 @@
+export * from './promised-region-info';
+
+/** The names of all (known) AWS regions */
+export const AwsRegionNames = [
+  'us-east-2',
+  'us-east-1',
+  'us-west-1',
+  'us-west-2',
+  'ap-south-1',
+  'ap-northeast-3',
+  'ap-northeast-2',
+  'ap-southeast-1',
+  'ap-southeast-2',
+  'ap-northeast-1',
+  'ca-central-1',
+  'cn-north-1',
+  'cn-northwest-1',
+  'eu-central-1',
+  'eu-west-1',
+  'eu-west-2',
+  'eu-west-3',
+  'eu-north-1',
+  'sa-east-1',
+];
+
+/** Possibly non-exaustive list of all service names, used to locate service principals */
+export const AwsServiceNames = [
+  'application-autoscaling',
+  'autoscaling',
+  'codedeploy',
+  'ec2',
+  'events',
+  'lambda',
+  'logs',
+  's3',
+  'sns',
+  'sqs',
+  'states',
```

Comment:
Same question...?

Reply:
No service principal in "built-in" data.